src/manifolds/HyperbolicHyperboloid.jl

```diff
@@ -1,3 +1,25 @@
+@doc raw"""
+    change_representer(M::Hyperbolic{n}, ::EuclideanMetric, p, X)
+
+Change a Eucliden gradient from the embedding that was already projected onto the tangent space at `p`.
+We only have to correct for the metric, which means that the sign of the last entry changes, since
+for the result ``Y``  we are looking for a tangent vector such that
+
+```math
+    g_p(Y,Z) = -y_{n+1}z_{n+1} + \sum_{i=1}^n y_iz_i = \sum_{i=1}^{n+1} z_ix_i
+```
+
+holds, which directly yields ``y_i=x_i`` for ``i=1,\ldots,n`` and ``y_{n+1}=-x_{n+1}``.
+"""
+change_representer(::Hyperbolic, ::EuclideanMetric, ::Any, ::Any)
+
+function change_representer!(M::Hyperbolic, Y, ::EuclideanMetric, p, X)
+    copyto!(M, Y, p, X)
+    Y[end] *= -1
+    return Y
+end
+
+# a metric change does not seem possible here
```

Comment:
It is indeed not possible due to [Sylvester's law of inertia](https://en.wikipedia.org/wiki/Sylvester%27s_law_of_inertia) . I think it would be fair to throw an error here.

Reply:
Yes, I think we could have something like
```julia
change_metric(::Hyperbolic, ::EuclideanMetric, ::Any, ::Any) = error("Changing metric from Euclidean to Hyperbolic is not possible (see Sylvester's law of inertia).")
```

Reply:
Sure I'll add that.